src/plugins/intel_npu/src/backend/src/zero_backend.cpp

```diff
@@ -30,7 +30,7 @@ uint32_t ZeroEngineBackend::getGraphExtVersion() const {
 }
 
 bool ZeroEngineBackend::isBatchingSupported() const {
-    return _instance->isExtensionSupported(std::string(ZE_GRAPH_EXT_NAME_1_6), ZE_MAKE_VERSION(1, 6));
```

Comment:
This needs to remain on ZE_GRAPH_EXT_NAME_1_6 for compatibility with old drivers

Reply:
Fixed, since the macro is not in current level-zero-ext commit, use hardcode now

Reply:
Looks good, cheers